app/controllers/foods_controller.rb

```diff
@@ -0,0 +1,6 @@
+class FoodsController < ApplicationController
+  def index
+    @search_results = FoodFacade.search(params[:q])
+    @total_results = FoodFacade.total_results(params[:q])
```

Comment:
You could look into using the following syntax:

@total_results, @search_results= FoodFacade.your_method(params[:q])

You could put this in your controller, and then have your_method return a 2d array where the first element was the totalHits, and the second element was the collection of food objects you created.

The call to get all the food data comes back with 'totalhits', so no need to call the facade again, which in then would call the service again.